plugins/notifications/ClickAppsSoundsNotify.qml

```diff
@@ -36,7 +37,7 @@ ItemPage {
         anchors {
             left: parent.left
             right: parent.right
-            top: parent.top
+            top: parent.header.bottom
```

Comment:
This creates a large gap between the header and top item:

![screenshot20181121_124235616](https://user-images.githubusercontent.com/21966173/48862332-b2c08a80-ed8b-11e8-8a53-c1b91df52122.png)